### PR TITLE
feat: Change proposal body limit and strategies limit for turbo spaces

### PR DESCRIPTION
**Pull request by ChaituVR**
Closes: https://github.com/snapshot-labs/snapshot/issues/4488

### Summary
Increase proposal body limit and strategies limit based on space type

- [ ] Merge only after https://github.com/snapshot-labs/snapshot-sequencer/pull/275

### How to test
- Use sequencer with https://github.com/snapshot-labs/snapshot-sequencer/pull/275 or wait until it is merged
- Add 10 strategies in space settings for turbo spaces
- Add 8 strategies in space settings for normal spaces
- Proposal body with 20000 characters for turbo spaces
- Proposal body with 10000 characters for normal spaces
- Update proposal with different chars length

### CODE REVIEW

**Comment by samuveth** [comment]
Please wait for my review on this :)

**Comment by bonustrack** [comment]
We should not limit the textarea (for both 10 and 20k limit), and instead show a red border or error message if you overflow. This is helpful for example when someone copy paste a longer text.

**Comment by ChaituVR** [comment]
> We should not limit the textarea (for both 10 and 20k limit), and instead show a red border or error message if you overflow. This is helpful for example when someone copy paste a longer text.

Existing functionality, will do a separate PR to avoid confusion

File: src/components/SettingsStrategiesBlock.vue [hunk 1]
```diff
@@ -2,16 +2,26 @@
 import { SpaceStrategy } from '@/helpers/interfaces';
 import { clone } from '@snapshot-labs/snapshot.js/src/utils';
 import schemas from '@snapshot-labs/snapshot.js/src/schemas';
+import spaceSchema from '@snapshot-labs/snapshot.js/src/schemas/space.json';
```
Comment (samuveth) [inline src/components/SettingsStrategiesBlock.vue:5]: This seems like a duplicate import. Instead I would create a var like this `const spaceSchema = schemas.space`

File: src/composables/useFormSpaceProposal.ts [hunk 2]
```diff
@@ -51,7 +51,7 @@ const userSelectedDateStart = ref(false);
 const userSelectedDateEnd = ref(false);
 const sourceProposalLoaded = ref(false);
 
-export function useFormSpaceProposal() {
+export function useFormSpaceProposal({ spaceType = 'default' } = {}) {
```
Comment (samuveth) [inline src/composables/useFormSpaceProposal.ts:54]:
```suggestion
export function useFormSpaceProposal({ spaceType = 'default' }) {
```
Doesn't seem to need `= {}` as `spaceType` is always defined

File: src/composables/useFormSpaceSettings.ts [hunk 3]
```diff
@@ -59,7 +59,10 @@ const formSettings = ref(clone(EMPTY_SPACE_FORM));
 const initialFormState = ref(clone(EMPTY_SPACE_FORM));
 const inputRefs = ref<any[]>([]);
 
-export function useFormSpaceSettings(context: 'setup' | 'settings') {
+export function useFormSpaceSettings(
+  context: 'setup' | 'settings',
+  { spaceType = 'default' } = {}
```
Comment (samuveth) [inline src/composables/useFormSpaceSettings.ts:64]: Same here, why do we need `= {}`?

**Review by samuveth — APPROVED** [review]
Left a few comments, otherwise looks good

tACK

File: src/composables/useFormSpaceSettings.ts (hunk 3 above)
Comment (ChaituVR) [reply src/composables/useFormSpaceSettings.ts:64]: There are some places where we don't pass these params to `useFormSpaceSettings` or `useFormSpaceProposal` composables. It is a bit painful to pass `spaceType` everywhere. 🙈 so this param will be `{}` by default.

Comment (samuveth) [reply src/composables/useFormSpaceSettings.ts:64]: But you already defined it here to be `default` always with `spaceType = 'default'`

Comment (samuveth) [reply src/composables/useFormSpaceSettings.ts:64]: If I'm not mistaken the object can never be just `{}`

Comment (samuveth) [reply src/composables/useFormSpaceSettings.ts:64]: Ok checked and seems to be needed. Still not sure I understand why tho :D

Comment (ChaituVR) [reply src/composables/useFormSpaceSettings.ts:64]: Yea bit confusing 😄

**Review by wa0x6e — APPROVED** [review]
utAck

**Pull request merged**